rpc/src/v1/types/account_info.rs

```diff
@@ -21,6 +22,28 @@ pub struct AccountInfo {
 	pub name: String,
 }
 
+/// Datastructure with proof for one single storage-entry
+#[derive(Debug, Default, Clone, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct StorageProof {
+	pub key: U256,
+	pub value: U256,
+	pub proof: Vec<Bytes>
+}
+
+/// Account information.
+#[derive(Debug, Default, Clone, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct EthAccount {
+	pub address: H160,
+	pub balance: U256,
+	pub nonce: U256,
+	pub code_hash: H256,
+	pub storage_hash: H256,
+	pub account_proof: Option<Vec<Bytes>>,
+	pub storage_proof: Option<Vec<StorageProof>>,
+}
```

Comment:
Doesn't make sense to me that `account_proof` is an Option because `EthAccount` is only constructed if `prove_account` returns `Some(proof, account)`?

Reply:
this is true, so I removed the option. Also for `storage_proof` since this will return an empty array if no keys are specified.